frontend/public/static/index.css

```diff
@@ -0,0 +1,11 @@
+html, body {
+    height: 100%;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+}
+
+div#root {
+    height: 100%;
+    width: 100%;
```

Comment:
Does the editor work without some of these values? The only reason why I ask is cause the HTML tag already has some styling applied to it in index.tsx (height set to 100vw) which you'd then have to delete once you copy this over to the styled-components form. Then again, I tested this locally and it seems the page height and all works fine, though I'm not sure whether that's the existing CSS overriding these new styles or vice versa.

Reply:
It looks like I can get rid of the width and padding/margin attributes. Though the HTML tag has height set, the body does not so I still added that line to the body tag.